.github/issue_template.md

```diff
@@ -1,22 +1,45 @@
-Please read this text and delete everything up to the line of dashes before
-filing your issue. Please
+<!--
+Thank you for opening issue.
+Please read this text and provide all necessary info. Please understand that without
+reproduce steps, clear explanations, and information about your particular setup 
+issue may be closed as unreproducible.
+Lines enclosed by arrows are comments, and will be deleted automatically.
+-->
 
-- fill out the table below the line of dashes and keep it at the end of your
-  issue text. The provided answers below are examples, only keep one of them or
-  replace them entirely with something else. But please provide an answer for
-  every line.
+<!-- Issue template:
+- First please provide clear explanations of issue type. What causing it,
+  and how we could test it.
+- Provide short and understandable steps to reproduce the issue in the list
```

Comment:
not a fan of these steps list. In my experience they are never provided in a useful way.